Tasks/SonarQubePostTest/SonarQubeHelper.ps1

```diff
@@ -0,0 +1,93 @@
+function InvokeMsBuildRunnerPostTest
+{
+	$bootstrapperPath = GetBoostrapperPath
+	$arguments = GetMsBuildRunnerPostTestArgs
+
+	Write-Verbose -Verbose "Executing $bootstrapperPath with arguments $arguments"
+	Invoke-BatchScript $bootstrapperPath -Arguments $arguments
+}
+
+function GetBoostrapperPath
+{
+	$bootstrapperPath = GetTaskContextVariable "bootstrapperPath" 
+
+	if (!$bootstrapperPath -or ![System.IO.File]::Exists($bootstrapperPath))
+	{
+		throw "The SonarQube MsBuild Runner executable could not be found. Does your build definition include the SonarQube Pre-Build step?"
```

Comment:
"MSBuild.SonarQube.Runner executable..."

"Check that the build definition includes..."